### PR TITLE
Deprecation index md formatter

### PR DESCRIPTION
**Pull request by BioPhoton**

### CODE REVIEW

File: packages/deprecation-crawler/src/lib/output-formatters/markdown/deprecation-index.md.formatter.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+  console.log('Updated group-based markdown format');
+}
+
+async function getDeprecationList(groupedDeprecations: {
+  [version: string]: Deprecation[];
+}): Promise<string> {
+  return Object.entries(groupedDeprecations)
+    .map(([version, deprecations]) => {
+      const treeName = deprecations[0].version;
```
Comment (timdeschryver) [inline packages/deprecation-crawler/src/lib/output-formatters/markdown/deprecation-index.md.formatter.ts:46]: Note that `version` can be untagged (empty).
In #84 this was fixed for another formatter where we did `deprecation.version || tag`.

File: packages/deprecation-crawler/src/lib/constants.ts [hunk 2]
```diff
@@ -15,3 +15,5 @@ export const enum CRAWLER_MODES {
   'SANDBOX' = 'SANDBOX_MODE',
   'CI' = 'CI',
 }
+export const MD_GROUP_OPENER = '<!-- ruid-groups';
```
Comment (timdeschryver) [inline packages/deprecation-crawler/src/lib/constants.ts:18]: Should [these](https://github.com/timdeschryver/deprecation-manager/blob/master/packages/deprecation-crawler/src/lib/output-formatters/markdown/generate-group-based-formatter.ts#L9-L10) be removed?

File: packages/deprecation-crawler/src/lib/output-formatters/markdown/deprecation-index.md.formatter.ts (hunk 1 above)
Comment (BioPhoton) [reply packages/deprecation-crawler/src/lib/output-formatters/markdown/deprecation-index.md.formatter.ts:47]: https://github.com/timdeschryver/deprecation-manager/pull/85/commits/d92cc2d91f23e4944cf167ffe58691567b745c10

File: packages/deprecation-crawler/src/lib/constants.ts (hunk 2 above)
Comment (BioPhoton) [reply packages/deprecation-crawler/src/lib/constants.ts:18]: done

**Pull request merged**